tests/ios/xcodeproj/Single-Static-Framework-Project.xcodeproj/bazelinstallers/install.sh

```diff
@@ -24,10 +24,14 @@ com.apple.product-type.framework.static)
     # We only care about the entry ending with TARGET_NAME" so that we can get the path to its directory
     QUERY=$(grep -A 2 important_output "$BAZEL_BUILD_EVENT_TEXT_FILENAME" | grep -w uri | grep ${TARGET_NAME}\" | sed "s/uri: \"file:\/\///")
     if [[ -z $QUERY ]]; then
-        # For virtual frameworks, this is disabled
-        exit 0
+        # For virtual frameworks, this is expected to be empty. Nothing to do.
```

Comment:
See the other comment - it shouldn't really install the framework to the directory. Perhaps in this step we'd remove them?

https://github.com/bazel-ios/rules_ios/pull/661/files#diff-a15f63f3a76f05dbdfb6a9b817d2d5b1dfd28aa94bafc362eca9bd53f0dd77e4R902

Reply:
That's probably an improvement from what I have if it works, will try to remove the problematic fmw paths here and see how it goes. The issue is that `XCBBuildService` creates these framework paths under `DerivedData` so I don't think it's our code creating them but will double check. 

Even if there's an empty `.framework` directory under `DerivedData` the `clang/swiftc` indexing invocation in Xcode will fail due to missing headers/modulemaps and ignore the VFS overlay flag in the same invocation. Honestly this might be a `clang` bug but not sure.

Will try to clean things up at this step and report back. It should indeed work.

Reply:
Btw I tried to clean things in this step but at least in the repro that I had that wasn't enough to fix the indexing invocations

Other state changes in Xcode apparently trigger adding the malformed fmws under DD. Will stick with the pre-build workaround for now.